libs/designsystem/src/lib/components/modal/services/modal.helper.ts

```diff
@@ -63,8 +63,7 @@ export class ModalHelper {
       backdropDismiss: config.flavor === 'compact' || config.interactWithBackground ? false : true,
       showBackdrop: !config.interactWithBackground,
       componentProps: { config: config },
-      swipeToClose: config.flavor != 'compact',
-      presentingElement: modalPresentingElement,
```

Comment:
Yeah we are able to remove all the presenting element logic. I think you're right that ModalHelper isn't intended to be used by others. It seems like the methods were public because the `registerPresentingElement` method was called in `app.component.ts` in the `ngAfterContentInit` lifecycle hook and it was also set in the cookbook example.

I have done as you suggested - preserved the methods and just log a message that says the method is deprecated.

Reply:
Nice. Would be great to create an issue for removing the method in the next major version.

Reply:
I'm on it! :)